neutrons/MagnetismReflectometer
# Reorganize tests

## Description of work:
Standardize the tests a bit more (rename some, reorganize according to tested source). Also rename the release notes file in accordance to other packages

Check all that apply:
- [x] no release notes necessary
- [x] Reorganized some tests

**References:**
- Links to IBM EWM items: [8489](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=8489)

## Manual test for the reviewer
In the terminal, activate the appropriate conda environment and run:
```
pytest -n 1 tests/
```
All tests should pass

## Check list for the reviewer
- [ ] [release notes](https://mr-reduction.readthedocs.io/en/latest/releases.html)
updated, or an explanation is provided as to why release notes are unnecessary
